### PR TITLE
[SIG-1648] - Filter by source

### PR DESCRIPTION
**Pull request by janjaap**
This PR adds the section 'Bron' to the filter form. The list of hardcoded sources has been moved from the meldingen form to the `definitions` folder for reuse.

### CODE REVIEW

File: src/signals/incident/definitions/wizard-step-1-beschrijf.js [hunk 1]
```diff
@@ -29,25 +36,7 @@ export default {
...
-            'E-mail – Stadsdeel': 'E-mail – Stadsdeel',
-            'Webcare – CCA': 'Webcare – CCA',
-            'Eigen organisatie': 'Eigen organisatie',
-            'Meldkamer burger/ondernemer': 'Meldkamer burger/ondernemer',
-            'Meldkamer Handhaver': 'Meldkamer Handhaver',
-            'Meldkamer Politie': 'Meldkamer Politie',
-            VerbeterDeBuurt: 'VerbeterDeBuurt',
-            Waarnemingenapp: 'Waarnemingenapp',
-          },
+          values: sourceValuesObj,
```
Comment (spaces-ga) [inline src/signals/incident/definitions/wizard-step-1-beschrijf.js]:
```js
{
  '': 'Vul bron in',
  ...sourceList,
}
```

File: src/signals/incident/definitions/wizard-step-1-beschrijf.js [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 import checkVisibility from '../services/check-visibility';
 import getStepControls from '../services/get-step-controls';
 
+const sourceValuesObj = {
+  '': 'Vul bron in',
+};
+sourceList.forEach(({ key, value }) => {
+  sourceValuesObj[key] = value;
+});
+
```
Comment (spaces-ga) [inline src/signals/incident/definitions/wizard-step-1-beschrijf.js]: See previous comment

File: src/signals/incident/definitions/wizard-step-1-beschrijf.js (hunk 1 above)
Comment (janjaap) [reply src/signals/incident/definitions/wizard-step-1-beschrijf.js]: Not entirely; `sourceValuesObj` should be an object. The imported value is an array of objects.

Comment (spaces-ga) [reply src/signals/incident/definitions/wizard-step-1-beschrijf.js]: You're right! my mistake 🤦‍♂

**Pull request merged**